e2e/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '2'
+services:
+
+  broker:
+    image: nats
+    expose:
+      - 4222
+      - 8222
+
+  node:
+    build:
+      context: ..
+      dockerfile: e2e/Dockerfile.node
+    depends_on:
+      - broker
+    privileged: true
+    cap_add:
+      - MKNOD
+      - NET_ADMIN
+    expose:
+      - 1194
+    command: "--data-dir=/mysterium --config-dir=/mysterium/server_config --runtime-dir=/mysterium --broker-address=broker"
+
+  client:
+    build:
+      context: ..
+      #this effectively reuses node image built earlier
+      dockerfile: e2e/Dockerfile.node
+    depends_on:
+      - broker
+      - node
+    privileged: true
+    cap_add:
+      - MKNOD
+      - NET_ADMIN
+    ports:
+      - 4052:4050
```

Comment:
Do we really have to take hosts port then running tests? What if tests are run in docker-compose's neighbour container?

Reply:
Possible, but a bit complicated - at the moment is simplest way to run tests agains full started up system with only *single* port exposed